MDANSE_GUI/Src/MDANSE_GUI/Tabs/TrajectoryTab.py

```diff
@@ -60,9 +60,13 @@ def load_trajectories(self):
             str(self.get_path("trajectory")),
             "HDF5 files, MDANSE or H5MD format (*.mdt *.h5);;H5MD files (*.h5);;All files (*)",
         )
+
+        loaded_files = []  # list to store loaded files for recent files tracking
+
         for fname in fnames[0]:
             self.load_trajectory(PurePath(fname))
             last_path = str(PurePath(os.path.split(fname)[0]))
+            loaded_files.append(str(PurePath(fname)))
         if fnames[0]:
             self.set_path("trajectory", str(PurePath(last_path)))
```

Comment:
This is already a string? 

```suggestion
            self.set_path("trajectory", last_path)
```